apps/api-extractor/src/ast/AstNamespace.ts

```diff
@@ -105,13 +105,10 @@ export class AstNamespace extends AstModule {
       // the JSDoc comment Node can be found.
       // If there is no parent or grandparent of this VariableDeclaration then
       // we do not know how to obtain the JSDoc comment.
```

Comment:
Damn...  This feature was broken by [https://github.com/Microsoft/web-build-tools/commit/4eca6649d9dd3cee6e8a2998400c201702f5e107](https://github.com/Microsoft/web-build-tools/commit/4eca6649d9dd3cee6e8a2998400c201702f5e107)

This entire file is deleted in AE7, so I guess it's not worth fixing